apps/widgets/src/features/AdminDashboard/WidgetInfo.tsx

```diff
@@ -0,0 +1,116 @@
+import type { FC } from 'react';
+import { StyleSheet } from 'react-native';
+import { type Maybe, type Widget, WidgetStatus } from '@walless/graphql';
```

Comment:
we need to stop mixing `type` and `value` import, make it 2 separate lines:
```typescript
import type { Maybe, Widget } from '@walless/graphql';
import { WidgetStatus } from '@walless/graphql';
```
Don't use `Maybe` type, it is for GraphQL, not for React/Typescript.